lib/API/JSON/v1/Subscribers.php

```diff
@@ -204,18 +258,18 @@ function subscribe($data = []) {
       }
     }
 
-    if (!empty($captcha['type']) && $captcha['type'] === Captcha::TYPE_RECAPTCHA && empty($data['recaptcha'])) {
+    if (!empty($captcha_settings['type']) && $captcha_settings['type'] === Captcha::TYPE_RECAPTCHA && empty($data['recaptcha'])) {
       return $this->badRequest([
         APIError::BAD_REQUEST => WPFunctions::get()->__('Please check the CAPTCHA.', 'mailpoet'),
       ]);
     }
 
-    if (!empty($captcha['type'])) {
-      if ($captcha['type'] === Captcha::TYPE_RECAPTCHA) {
+    if (!empty($captcha_settings['type'])) {
```

Comment:
Using fast termination here would make this method a bit more readable by removing one level of nesting in conditions.
```
if (empty($captcha_settings['type'])) {
  return true;
}
```
You can also put the condition at the beginning of the `validateCaptcha` method so that it doesn't have to be repeated in each condition.